src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,55 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+
+    public static void provideEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.sendKeys(email);
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.sendKeys(password);
+    }
+
+    public static void clickSubmit() {
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type = 'submit']"));
+        submitButton.click();
+    }
+
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+
+    public static void provideEmail(String email) {
+        WebElement emailField = driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.sendKeys(email);
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.sendKeys(password);
+    }
+
+    public static void clickSubmit() {
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type = 'submit']"));
+        submitButton.click();
+    }
+
+    public void logIn() {
+        provideEmail("janezelenova@gmail.com");
+        providePassword("Floridaliving2023$");
+
+        //AND user clicks a submit button
+
+        clickSubmit();
+    }
+
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/224534534-c3a0e599-2a09-4ddb-8caa-df374b893be1.png)